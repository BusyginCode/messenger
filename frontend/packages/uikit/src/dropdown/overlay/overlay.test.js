import React from 'react'
import { shallow } from 'enzyme'
import Overlay from './overlay'

describe('Overlay component', () => {
  it('should render correctly', () => {
    expect(shallow(<Overlay options={[{ id: 1 }]} />)).toMatchSnapshot()
  })

  it('should render without option id correctly', () => {
    expect(shallow(<Overlay options={[{ children: 'text' }]} />)).toMatchSnapshot()
  })
})
