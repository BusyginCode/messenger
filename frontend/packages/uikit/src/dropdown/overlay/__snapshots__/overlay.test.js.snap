// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay component should render correctly 1`] = `
<div>
  <div
    className="item"
    key="1"
  />
</div>
`;

exports[`Overlay component should render without option id correctly 1`] = `
<div>
  <div
    className="item"
    key="0"
  >
    text
  </div>
</div>
`;
